ui/components/src/Legend/CompactLegend.tsx

```diff
@@ -26,7 +26,7 @@ interface CompactLegendProps {
  */
 export function CompactLegend({ height, items }: CompactLegendProps) {
   return (
-    <Box sx={{ width: '100%', height, paddingTop: 1, overflowY: 'scroll' }}>
+    <Box component="ul" sx={{ width: '100%', height, padding: [0, 1, 0, 0], overflowY: 'scroll', margin: 0 }}>
```

Comment:
I noticed while I was debugging that there were some `li`s that weren't parented by an `ol`, `ul`, or `menu` ([required by the spec](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li). Adjusting this `Box` to behave as a `ul` and then modifying the padding to continue looking like it did before.